### DISCUSSION TITLE
Fullstack.Net- Contexto.update ou updateasync

### DISCUSSION THREAD

**Discussion by suarezrafael** [balta-io/forum#595] [Tira Dúvidas] [answered]
Vídeo atualizando a categoria handler. No curso de Fullstack ao criar o handler de update de categoria foi utilizado o método update  do contexto. E em seguida o savechangesasync. Qual seria a lógica para usar o síncrono e depois salvar assíncrono? Fiquei com essa dúvida.

**Comment by andrebaltieri** [comment, marked as answer]
O método Update não tem Async pois ele simplesmente atualiza um objeto que já está no contexto.
Uma vez async, sempre procuramos usar o máximo de asyncs possíveis no método!
